source/SVInt.cpp

```diff
@@ -561,6 +570,68 @@ SVInt SVInt::replicate(const SVInt& times) const {
     return concatenate(ArrayRef<SVInt>(buffer.begin(), buffer.end()));
 }
 
+SVInt SVInt::bitSelect(int16_t lsb, int16_t msb) const {
+    if (msb < lsb) {
```

Comment:
I think so: I think that the only cases where this could happen we should (but obviously don't now) detect it before it gets here.